core/src/com/example/simulation/PathTile.java

```diff
@@ -0,0 +1,41 @@
+package com.example.simulation;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class PathTile extends Tile {
+
+    PathTile prev;
+    PathTile next;
+    List<Enemy> enemies;
+
+    protected PathTile copy() {
+        return new PathTile(getPosition().x, getPosition().y, copyList(enemies));
+    }
+
+    PathTile(int x, int y) {
+        super(x, y);
+    }
+
+    PathTile(int x, int y, List<Enemy> enemies){
+        super(x,y);
+        this.enemies = enemies;
+    }
+
+    public void setNext(PathTile next) {
+        this.next = next;
+        if (next != null)
+            next.prev = this;
+    }
+    
+    private List<Enemy> copyList(List<Enemy> enemies){
+        List<Enemy> newEnemyList = new ArrayList<>();
```

Comment:
Gegner vom gleichen Typ sind ja ohnehin gruppiert zu einem Gegner.
Reicht da dann nicht ein normaler Array, da wir ja die Anzahl der Gegnertypen kennen?

Reply:
War mir in dem Moment nicht ganz sicher, wenn wir aber nur einen Gegner auf einem Tile zulassen braucht man da entweder keine Liste und gruppiert schon beim Bewegen. Dann bräuchte man nur eine einzige Referenz auf einen Gegner im PathTile.
Ansonsten würde ein normaler Array reichen